debian/check_capstone.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Usage: ./check_capstone_pc.sh <path_to_deb_file> <expected_version>
+
+DEB_FILE=$1
+EXPECTED_VERSION=$2
+
+# Check if the deb file exists
+if [[ ! -f "$DEB_FILE" ]]; then
+  echo "Debian package file not found!"
+  exit 1
+fi
+
+# Create a temporary directory to extract the deb file
+TEMP_DIR=$(mktemp -d)
+
+# Extract the deb file
+dpkg-deb -x "$DEB_FILE" "$TEMP_DIR"
+
+# Path to the capstone.pc file
+CAPSTONE_PC="$TEMP_DIR/usr/local/lib/pkgconfig/capstone.pc"
+
+# Check if the capstone.pc file exists
+if [[ ! -f "$CAPSTONE_PC" ]]; then
+  echo "capstone.pc file not found in the package!"
+  rm -rf "$TEMP_DIR"
+  exit 1
+fi
+
+# Remove leading 'v' if present, e. g. v1.5.1 -> 1.5.1
+if [[ "$EXPECTED_VERSION" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    EXPECTED_VERSION=${EXPECTED_VERSION:1}
+fi
+
+# Check if the version follows the format X.Y.Z, e. g. 1.5.1 or 1.9.1
+if [[ ! "$EXPECTED_VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+    echo "ERROR: Version must be in the format X.Y.Z"
+    exit 1
+fi
+
+
+# Check the version in the capstone.pc file
+ACTUAL_VERSION=$(grep "^Version:" "$CAPSTONE_PC" | awk '{print $2}')
+if [[ "$ACTUAL_VERSION" != "$EXPECTED_VERSION" ]]; then
+  echo "Version mismatch! Expected: $EXPECTED_VERSION, Found: $ACTUAL_VERSION"
+  rm -rf "$TEMP_DIR"
+  exit 1
+fi
+
+# Check if the archs field is present in the capstone.pc file
+if grep -q "^archs=" "$CAPSTONE_PC"; then
+  echo "archs field found in capstone.pc file!"
+  rm -rf "$TEMP_DIR"
+  exit 1
+fi
+
```

Comment:
```suggestion
```

Sorry, I meant you can remove `archs=@CAPSTONE_ARCHITECTURES@` from `capstone.pc.in`. Because it isn't used.